core/profiling/eventpipe/README.md

```diff
@@ -0,0 +1,21 @@
+# .Net Core EventPipe profiler
```

Comment:
If this is intended to be viewed from the samples browser, it needs some metadata.

Reply:
See for example https://github.com/dotnet/samples/blame/f29a4a287dc428dab7891bc79ba8299d454d1d4d/core/diagnostics/DiagnosticScenarios/readme.md#L1-L10

Reply:
From searching through the samples repo, it looks like the metadata is only currently used for C#/VB samples. It's not clear to me if I should add it for a native project